applications/matter_bridge/pm_static_nrf5340dk_nrf5340_cpuapp nrf70ek.yml

```diff
@@ -0,0 +1,80 @@
+mcuboot:
```

Comment:
Two things:
1. You have changed the pm_static name, by giving it a suffix which means it will be selected only if prj.conf with such suffix will be selected. And such prj_nrf70ek.conf does not exist. You have to create it, and please make sure that all related ipc configuration files are pulled correctly.
2. You have to describe the new prj.conf and a way of building nrf70ek+nrf5340dk in the readme.
3. You have a blank space in the pm_static name, which will certainly lead into problems :)